pageserver/src/page_service.rs

```diff
@@ -784,6 +787,20 @@ impl PageServerHandler {
                 &ctx,
             )
             .await?;
+        } else {
+            let writer = pgb.copyout_writer();
+            let mut encoder = GzipEncoder::new(writer);
+            basebackup::send_basebackup_tarball(
```

Comment:
How will this work with old computes that do not expect a compressed basebackup? I recall that you mentioned some version negotiation, but I don't see it here

Or `tar::Archive::unpack` will take care of it automatically?

Reply:
Why do we need to spawn old computes?
I ca understand that we may need to support work with old computes one new pageserver is restarted.
When new pageserver have to spawn old compute? This basebackup is used only for endpoint startup.

Reply:
We have computes of old versions running for months and they could be restarted by k8s due to various reasons (k8s node went down, OOM, rescheduling, etc), so they may need to start again and get basebackup. We only start new computes with the newest possible version, yes

Reply:
Even with `imagePullPolicy=Always` and `:latest` tag?